gnmi_server/server.go

```diff
@@ -514,3 +526,56 @@ func (s *Server) Capabilities(ctx context.Context, req *gnmipb.CapabilityRequest
 		GNMIVersion:        "0.7.0",
 		Extension:          exts}, nil
 }
+
+type uint128 struct {
+	High uint64
+	Low  uint64
+}
+
+func (lh *uint128) Compare(rh *uint128) int {
```

Comment:
Check if lh or rh are nil at the start of Compare?

Reply:
rh nil check is needed. If lh is nil, the error should be handled when the function get called.